app/javascript/components/Link/index.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react"
+import classNames from "classnames"
+import "./index.scss"
+
+export enum LinkSize {
+  ExtraSmall = "xs",
+  Small = "sm",
+  Base = "bs",
+}
+
+export enum LinkVariant {
+  Neutral = "neutral",
+  NeutralLight = "neutral-light",
+  NeutralDark = "neutral-dark",
+  Info = "info",
+}
+
+export interface LinkProps extends React.HTMLAttributes<HTMLAnchorElement> {
+  variant: LinkVariant
+  text: string
+  size: LinkSize
+  underline?: boolean
+}
+
+export default function Link(props: LinkProps): JSX.Element {
```

Comment:
It is here https://github.com/abtion/muffi/pull/967/files#diff-1a37e8d8224a1fd673f4d0348683596e4f6b77dd3e4380b5d32b1a504ab8148eR40, however @jeppester is suggesting that maybe we shouldnt have a react component for Links https://github.com/abtion/muffi/pull/967#discussion_r885238173 and from using them in this project I would agree, it is quite painful process to use it

Reply:
After having given this a some extra thought I guess I think the link component is fine to have - for the sake of consistency.

But we should make it clear in the muffibook examples where the styling originates from, so that projects can choose to just use tailwind classes instead.

For Muffi I don't think we need link-component - or we can make a normal `a`-tag behave like it's meant to according to the design.

Reply:
I decided to slightly alter the link-component so that it defaults to the most commonly used color.
Then the only class we need to add is "Link", and I think that's fine.